src/table.mjs

```diff
@@ -248,20 +248,48 @@ export function makeQueryTemplate(operations, source) {
   ];
   for (let i = 0; i < filter.length; ++i) {
     appendSql(i ? `\nAND ` : `\nWHERE `, args);
+    if (!filter[i].operands) throw new Error("missing operands");
+    filter[i].operands.forEach((op) => {
+      if (op.type === "column") op.value = escaper(op.value);
```

Comment:
This is mutating the passed-in _operations_ object. I’d guess that’s not harmful under the current usage, but isn’t something we should do in general since in theory that operations object could be reused by the caller for something else. For example if you called makeQueryTemplate twice with the same _operations_ object, the column name would get double-escaped.

So instead of mutating _op_ here, we should do the escaping as needed when interpolating into a SQL string.